web/src/components/onboarding/OnboardingTaxId.tsx

```diff
@@ -46,6 +71,8 @@ export const OnboardingTaxId = ({
   const [locationValue, setLocationValue] = useState(state.profileData[fieldName]?.trim().slice(9, 12) ?? "");
   const [taxIdValue, setTaxIdValue] = useState(state.profileData[fieldName]?.trim().slice(0, 9) ?? "");
   const [errorMap, setErrorMap] = useState<ErrorMapType>({ taxId: false, taxIdLocation: false });
+  const initialDisplayType = useRef<"MASKED" | "PLAIN">(getInitialDisplayType());
```

Comment:
why are we tracking this value separately from the `displayTaxId` using a `useRef`? It seems like the only place this is used is to set the displayTaxId state?